lib/helpers/ContentUiHelper.ts

```diff
@@ -1754,132 +1635,121 @@ export class ContentUiHelper extends UiBaseLocators {
     } else {
       expect(count, `Expected only one element, but found ${count}`).toBe(0);
     }
-    await expect(this.refListBlock).toBeVisible({visible: isVisible});
+    await this.isVisible(this.refListBlock, isVisible);
   }
 
   async doesBlockCustomViewHaveValue(customBlockViewLocator: string, valueText: string) {
     const locator = this.page.locator(`${customBlockViewLocator} p`);
-    await expect(locator).toBeVisible();
-    await expect(locator).toHaveText(valueText);
+    await this.waitForVisible(locator);
+    await this.hasText(locator, valueText);
   }
-  
+
   async clickPropertyActionWithName(name: string) {
-    const actionLocator = this.propertyActionMenu.locator('umb-property-action uui-menu-item[label="' + name + '"]');
-    await expect(actionLocator).toBeVisible();
-    await actionLocator.click();
+    const actionLocator = this.propertyActionMenu.locator(`umb-property-action uui-menu-item[label="${name}"]`);
+    await this.click(actionLocator);
   }
   
   async isContentWithNameVisibleInList(contentName: string, isVisible: boolean = true) {
-    await expect(this.documentTableColumnName.filter({hasText: contentName})).toBeVisible({visible: isVisible});
+    await this.isVisible(this.documentTableColumnName.filter({hasText: contentName}), isVisible);
   }
   
   async selectDocumentBlueprintWithName(blueprintName: string) {
-    await expect(this.documentCreateOptionsModal.locator('uui-menu-item', {hasText: blueprintName})).toBeVisible();
-    await this.documentCreateOptionsModal.locator('uui-menu-item', {hasText: blueprintName}).click();
+    await this.click(this.documentCreateOptionsModal.locator('uui-menu-item', {hasText: blueprintName}));
```

Comment:
In line 1648 we define a locator as a constant, but in line 1657 we dont, Where is the line for defining a locator as a const instead of just inserting it into the click action?